apps/client/src/assets/i18n/en.json

```diff
@@ -587,9 +587,9 @@
     "Amount_craftable": "You can craft {{amount}}",
     "Add_item": "Add an item",
     "List_large": "This is a large list",
-    "Enable_performance_mode": "You might want to enable performance mode in settings in order to get faster rendering",
+    "Enable_performance_mode": "You might want to enable performance mode in settings in order to get faster rendering. Do not report performance issues displaying this error as a bug, it will not be supported.",
     "List_too_large": "List too large",
-    "List_too_large_description": "This list is too large and might hit database size limitations because of all the informations it carries. You aren't allowed to add more items to make sure you won't lose the list.",
+    "List_too_large_description": "This list is too large and might hit database size limitations because of all the information it carries. You aren't allowed to add more items to make sure you won't lose the list.",
```

Comment:
The last sentence here kinda sounds like you're saying the user isn't allowed to lose the list. Maybe simplify like this:
> This list is too large and might slow down your computer or hit database limitations. To ensure you won't lose the list, you are no longer able to add further items to it unless existing items are removed.